tests/test_base.py

```diff
@@ -694,6 +694,32 @@ def test_base_index_access():
         h[0:10:20j, 0:5:10j, "hello", False, 5]
 
 
+def test_base_proxy():
+    """
+        Test base proxy -- whether BaseHist proxy works properly.
+    """
+    h = BaseHist.Regular(10, 0, 1, name="x").fill([0.5, 0.5])
+    assert h[0.5j] == 2
+
+    h = (
+        BaseHist()
+        .Regular(10, 0, 1, name="x")
+        .Regular(10, -1, 1, name="y")
+        .fill([0.5, 0.5], [-0.2, 0.6])
+    )
+
+    assert h[0.5j, -0.2j] == h[bh.loc(0.5), bh.loc(0.6)] == 1
```

Comment:
Please write asserts 1 per line, with the "answer" on each line. That way, reporting doesn't spit way to much text out when there's a failure. And it's easier to read.

```suggestion
    assert h[0.5j, -0.2j] == 1
    assert h[bh.loc(0.5), bh.loc(0.6)] == 1
```

Reply:
Okay.